google-cloud-storage/samples/acceptance/acls_test.rb

```diff
@@ -18,7 +18,7 @@
 describe "ACL Snippets" do
   let(:storage_client)   { Google::Cloud::Storage.new }
   let(:local_file)       { File.expand_path "data/file.txt", __dir__ }
-  let(:email)            { "user-test@test.com" }
+  let(:email)            { "user-blowmage@gmail.com" }
```

Comment:
We should try not to use Mike's email in tests anymore. Can we update the fixtures in the console instead?

Reply:
Sure @dazuma , I'll fix it in a separate PR because this PR is not causing the issue.